modules/cannarithm/include/opencv2/cann.hpp

```diff
@@ -0,0 +1,335 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_CANN_HPP
```

Comment:
Identifier should have the following format: `OPENCV_<module>_<header_underscrore_subpath>_HPP`

Reply:
Thanks for the code review, will fix them in next commit.

Reply:
Done. change all identifiers in hpp.